qa-test-app/src/main/res/layout/components_activity_layout.xml

```diff
@@ -29,14 +29,30 @@
         android:text="Start Navigation"
         android:background="@color/primary" />
 
-    <com.mapbox.navigation.ui.voice.view.MapboxAudioGuidanceButton
-        android:id="@+id/soundButton"
+    <androidx.appcompat.widget.LinearLayoutCompat
```

Comment:
Looking at this change I understand that the plan is to keep adding more buttons to this linear layout as they come in correct? For ex: the next button will be `CameraModeButton`

@tomaszrybakiewicz

Reply:
Resolving based on https://github.com/mapbox/mapbox-navigation-android/pull/6158#issuecomment-1217357778